.github/workflows/ci.yml

```diff
@@ -36,12 +37,12 @@ jobs:
                 python-version: "3.x"
         -   name: "Main Script"
             run: |
-                export SETUPTOOLS_USE_DISTUTILS=stdlib
-
-                EXTRA_INSTALL="numpy mako scipy"
-                export PYFMMLIB_BUILD_MODE=setuptools
+                export FC=gfortran-12
                 export F77=gfortran-12
                 export F90=gfortran-12
+
+                PROJECT_INSTALL_FLAGS="--verbose --no-build-isolation"
```

Comment:
What's behind the `--no-build-isolation`?

Reply:
It was recommended in the `meson-python` docs:
https://meson-python.readthedocs.io/en/latest/how-to-guides/editable-installs.html
It should just not create a separate venv for the build, I guess? They do some tricks to make the editable thing work..